2_base_and_sensor.ipynb

```diff
@@ -89,13 +174,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "これは、1m/秒の速度で直進することを意味します。\n",
     "\n",
     "台車を後進させるにはマイナスの指令値を入力します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Above command means going straight at a speed of 1 m/sec.\n",
```

Comment:
"The above command means: go forward at a speed of 1 m/sec.\n",

If there is actually more than one command, then use this:

"The above commands mean: go forward at a speed of 1 m/sec.\n",